infrastructure/json/src/main/java/tech/pegasys/teku/infrastructure/json/types/ByteTypeDefinition.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2021 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.infrastructure.json.types;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.JsonParser;
+import java.io.IOException;
+
+class ByteTypeDefinition implements DeserializableTypeDefinition<Byte> {
+
+  @Override
+  public void serializeOpenApiType(final JsonGenerator gen) throws IOException {
+    gen.writeStartObject();
+    gen.writeStringField("type", "number");
+    gen.writeStringField("format", "byte");
```

Comment:
number? i thought bytes serialized to string as hex...

Reply:
I'm unclear what the right answer for this one is - This is for a single byte, not a byte array (which does serialise as hex).  I don't think we actually use it anywhere currently so there isn't an existing example I'm aware of to check against.

Reply:
if it serializes to hex, then its string... unless we're not quoting